README.md

```diff
@@ -40,7 +40,7 @@ expo install sentry-expo
 Add the following to your app's main file (usually `App.js`):
 
 ```js
-import * as Sentry from 'sentry-expo';
+import { Sentry } from 'sentry-expo';
```

Comment:
hm... I'm not crazy about having different import syntax than `@sentry/react-native` https://github.com/getsentry/sentry-react-native. Just in case people refer to those docs (which I think is quite common when using `sentry-expo`. 

I think the best scenario here would be:

```
import * as Sentry from 'sentry-expo';

Sentry.init() // universal: web, android, and ios

Sentry.<directly access any native functionality>

Sentry.Browser.<access any web functionality>
```

what do you think?

Reply:
Yes this would be ideal and I think what we were expecting with #165

Reply:
I'll have to re-think this with the latest repo updates.

Reply:
Mainly how we could make `init` universal with different options. Like would you have to specify type `Sentry.init("web")` (just an example)